examples/stargate-prometheus-grafana/cqlsh-commands.txt

```diff
@@ -0,0 +1,5 @@
+backend-1
+docker run --rm -it nuvo/docker-cqlsh cqlsh host.docker.internal 9044 --cqlversion=3.4.4
```

Comment:
We need to set cqlversion on these?

Reply:
Yes, I found that I had to specify or it failed.